cardano-db-sync/cardano-db-sync.cabal

```diff
@@ -41,6 +41,7 @@ library
                         Cardano.DbSync.Plugin
                         Cardano.DbSync.Plugin.Default
                         Cardano.DbSync.Plugin.Default.Insert
+                        Cardano.DbSync.Plugin.Default.InsertShelley
```

Comment:
Would much prefer `Cardano.DbSync.Plugin.Default.Byron.Insert` and `Cardano.DbSync.Plugin.Default.Shelley.Insert`.

Reply:
The idea was to remove `Default` since I'm not sure what I'm defaulting to and replace that with specific implementations of `Byron` and `Shelley`.

Reply:
The are currently two extecutables in this repo, `cardano-db-sync` and `cardano-db-sync-extended`. The former uses this `Default` version of the functionality and the latter uses `Default` as well as the functionality in `Cardano.DbSync.Plugin.Epoch`.